assembly/gradle-plugin/src/main/java/org/eclipse/sw360/antenna/frontend/gradle/AntennaGradleTask.java

```diff
@@ -14,6 +15,6 @@
 
 public class AntennaGradleTask extends AnalyzeTask {
     protected String getPluginDescendantArtifactIdName() {
-        return "antenna-maven-plugin";
+        return "gradle-plugin";
```

Comment:
Is this renaming necessary as part of the Gradle to Maven conversion? If so, why?

Reply:
I'll isolate this change into a second commit which does the name fix.